src/data-workspace/inputs/option-set.js

```diff
@@ -81,22 +118,22 @@ export const OptionSet = ({
                     onKeyDown={onKeyDown}
                     onBlur={() => input.onBlur()}
                     disabled={disabled || locked}
+                    inputMaxHeight={'24px'}
                 >
                     {options.map(({ id, code, displayName }) => (
-                        <SingleSelectOption
+                        <SelectOptionComponent
                             key={id}
                             label={displayName}
                             value={code}
                         />
                     ))}
-                </SingleSelect>
+                </SelectComponent>
             </div>
             {input.value && (
                 <Button
                     small
                     secondary
                     className={styles.whiteButton}
-                    {...input}
```

Comment:
Not sure why this was needed?

Reply:
Was it to pass `name` to the Button? Or it might have just been there because we were generally passing all the input props down in the other input types?

Reply:
I can add the name. `focus` styling are working. Not sure what functionality we would want for `onFocus` and `onBlur` on this button?